server/database-postgres/models.js

```diff
@@ -0,0 +1,84 @@
+const Sequelize = require('sequelize');
+const { DataTypes } = require('sequelize');
+
+const db = new Sequelize({
+  dialect: 'postgres',
+  user: 'postgres',
+  password: 'Password123!',
+  database: 'users1',
+  port: 5432,
+  host: 'localhost',
+  logging: false
+});
+
+const User = db.define('user', {
+  userId: {
+    allowNull: false,
+    autoIncrement: true,
+    unique: true,
+    primaryKey: true,
+    type: DataTypes.INTEGER
+  },
+  name: {
+    allowNull: false,
+    type: DataTypes.STRING
+  },
+  bio: {
+    allowNull: false,
+    type: DataTypes.TEXT
+  },
+  joinDate: {
+    allowNull: false,
+    type: DataTypes.DATE
+  },
+  avatarUrl: {
+    allowNull: false,
+    type: DataTypes.STRING
+  },
+  isSuperhost: {
+    allowNull: false,
+    type: DataTypes.BOOLEAN
+  },
+  identityVerified: {
+    allowNull: false,
+    type: DataTypes.BOOLEAN
+  },
+  responseRate: {
+    allowNull: false,
+    type: DataTypes.INTEGER
+  },
+  responseTime: {
+    allowNull: false,
+    type: DataTypes.STRING
+  }
+});
+
+const Language = db.define('language', {
+  languageId: {
+    allowNull: false,
+    autoIncrement: true,
+    primaryKey: true,
+    type: DataTypes.INTEGER
+  },
+  name: {
+    allowNull: false,
+    type: DataTypes.STRING,
+  }
+});
+
+const UserLanguage = db.define('user_language', {})
+
+UserLanguage.belongsTo(User);
+UserLanguage.belongsTo(Language);
+
+const sync = () => {
+  return db.sync({ force: true });
```

Comment:
Careful here! passing `force: true` to options object in `.sync` will run a `DROP TABLE IF EXISTS` everytime you start up your app.  I accidentally wiped my 10M records this way :(

Reply:
yeah, i already did wipe my old one out lol. will change it